canonical-web-and-design/vanilla-framework
# Search updates for top navigation

## Done

Update search for new nav (mobile search dropdown)

Fixes #4376 

## QA

- Open [demo](https://vanilla-framework-4396.demos.haus/docs/examples/patterns/navigation/search-dark)
- Make sure the expanding search works as expected
  - check if search can be toggled on/off
  - check if it works as expected on both mobile and desktop
  - check light and dark themes
    - https://vanilla-framework-4396.demos.haus/docs/examples/patterns/navigation/search-light
    - https://vanilla-framework-4396.demos.haus/docs/examples/patterns/navigation/search-dark
  - check deprecated example to make sure it still works fine
    - https://vanilla-framework-4396.demos.haus/docs/examples/patterns/navigation/deprecated
- Review updated documentation:
  - https://vanilla-framework-4396.demos.haus/docs/patterns/navigation#expanding-search-box


## Screenshots

![new-nav](https://user-images.githubusercontent.com/83575/162771809-07741310-1833-4f4d-8f91-5c6440aa75f9.gif)
